keyboards/jukaie/jk01/keymaps/default/keymap.c

```diff
@@ -0,0 +1,134 @@
+/* Copyright 2023 JUKAIE <lice@jukaie.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+
+enum my_keycodes {
+    KC_MC = QK_USER,
+    KC_SIRI,
+    KC_DND,
+    KC_SPOT,
+    KC_EJ,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+
+    switch (keycode) {
+
+        case KC_MC:
+            if (record->event.pressed) {
+                host_consumer_send(0x29F);
+            } else {
+                host_consumer_send(0);
+            }
+            return false; /* Skip all further processing of this key */
```

Comment:
This functionality is already in core, as `KC_MISSION_CONTROL`
https://docs.qmk.fm/#/keycodes_basic?id=media-keys